src/olympia/scanners/models.py

```diff
@@ -94,6 +94,24 @@ def get_scanner_name(self):
     def get_pretty_results(self):
         return json.dumps(self.results, indent=2)
 
+    def get_files_by_matched_rules(self):
+        res = dict()
+        if self.scanner is YARA:
+            for item in self.results:
+                if item['rule'] not in res:
+                    res[item['rule']] = []
+                res[item['rule']].append(item['meta']['filename'])
+        elif self.scanner is CUSTOMS:
+            scanMap = (self.results['scanMap'] if 'scanMap' in self.results
```

Comment:
`scanMap = self.results.get('scanMap', {})`

Reply:
ha, I meant to change that, thanks!